src/ncp/posix/netif.cpp

```diff
@@ -202,6 +202,23 @@ void Netif::SetNetifState(bool aState)
     }
 }
 
+void Netif::Ip6Receive(const uint8_t *aBuf, uint16_t aLen)
+{
+    otbrError error = OTBR_ERROR_NONE;
+
+    VerifyOrExit(aLen <= kIp6Mtu, error = OTBR_ERROR_DROPPED);
+    VerifyOrExit(mTunFd > 0, error = OTBR_ERROR_INVALID_ARGS);
+
+    otbrLogInfo("Packet from NCP (%u bytes)", static_cast<uint16_t>(aLen));
```

Comment:
Can we remove `static_cast<uint16_t>()` as it is already `uint16_t aLen`?
 ```suggestion
    otbrLogInfo("Packet from NCP (%u bytes)", aLen);
```